scripts/ci/gitlab-ci/run.sh

```diff
@@ -1,48 +1,63 @@
 #!/bin/bash --login
-
-if [ -n "${GITLAB_SITE}" ]
-then
-  export CI_SITE_NAME="${GITLAB_SITE}"
-else
-  export CI_SITE_NAME="GitLab CI"
-fi
+set -ex
 
 export CI_BUILD_NAME="${CI_COMMIT_BRANCH#github/}_${CI_JOB_NAME}"
-export CI_SOURCE_DIR="${CI_PROJECT_DIR}"
-export CI_ROOT_DIR="${CI_PROJECT_DIR}/.."
-export CI_BIN_DIR="${CI_ROOT_DIR}/${CI_BUILD_NAME}"
 export CI_COMMIT_REF=${CI_COMMIT_SHA}
+export CI_ROOT_DIR="${CI_PROJECT_DIR}/.."
+export CI_SITE_NAME="${GITLAB_SITE}"
+export CI_SOURCE_DIR="${CI_PROJECT_DIR}"
 
-STEP=$1
-CTEST_SCRIPT=scripts/ci/cmake/ci-${CI_JOB_NAME}.cmake
+export CI_BIN_DIR="${CI_ROOT_DIR}/${CI_BUILD_NAME}"
 
-# Update and Test steps enable an extra step
-CTEST_STEP_ARGS=""
-case ${STEP} in
-  test) CTEST_STEP_ARGS="${CTEST_STEP_ARGS} -Ddashboard_do_end=ON" ;;
-esac
-CTEST_STEP_ARGS="${CTEST_STEP_ARGS} -Ddashboard_do_${STEP}=ON"
+readonly CTEST_SCRIPT=scripts/ci/cmake-v2/ci-${CI_JOB_NAME}.cmake
+if [ ! -f "$CTEST_SCRIPT" ]
+then
+  echo "[E] Variable files does not exits: $CTEST_SCRIPT"
+  exit 1
+fi
 
-if [ -n "${CMAKE_ENV_MODULE}" ]
+readonly STEP=$1
+if [ -z "$STEP" ]
 then
-  module load ${CMAKE_ENV_MODULE}
+  echo "[E] No argument given: $*"
+  exit 2
+fi
 
-  echo "**********module avail Begin************"
-  module avail
-  echo "**********module avail End**************"
+if [ "$STEP" = "update" ]
+then
+  echo "**********Env Begin**********"
+  env | sort
```

Comment:
Are you thinking to leave this in?  I didn't scan carefully through the printed environment variables, but is it possible some sensitive information is exposed there?

Reply:
That is correct, this was inherited from existing code, if we ever need this we can re-add it later.